perception/radar_tracks_msgs_converter/src/radar_tracks_msgs_converter_node/radar_tracks_msgs_converter_node.cpp

```diff
@@ -162,7 +162,7 @@ TrackedObjects RadarTracksMsgsConverterNode::convertRadarTrackToTrackedObjects()
   TrackedObjects tracked_objects;
   tracked_objects.header = radar_data_->header;
   tracked_objects.header.frame_id = node_param_.new_frame_id;
-  using P_IDX = tier4_autoware_utils::pose_covariance_index::POSE_COV_IDX;
+  using C_IDX = tier4_autoware_utils::xyzrpy_covariance_index::XYZRPY_COV_IDX;
   using R_IDX = tier4_autoware_utils::xyz_upper_covariance_index::XYZ_UPPER_COV_IDX;
```

Comment:
What are `C_` and `R_`?

Reply:
I mean Covariance index and Radar covariance index, but I think it is not suitable abbreviation.
Do you have any suggestion?

Reply:
I'd use `POSE_IDX`, `RADER_IDX` or `POSE_COV_IDX`, `RADER_COV_IDX`.
And rename variables:
- covariance -> pose_cov
- radar_track.position_covariance -> radar_cov